.github/workflows/pr-screener.yml

```diff
@@ -3,7 +3,7 @@ on:
   pull_request:
     branches: [master]
 jobs:
-  screener:
+  screen:
```

Comment:
Nitpick: this could be renamed to express what the check does, no? `screenshot_tests` or something like that.

Reply:
sure